lib/tasks/comments.rake

```diff
@@ -0,0 +1,21 @@
+# frozen_string_literal: true
+
+desc 'Converting data from string to integer for comments status'
+
+namespace :comments do
+  task convert_string_status_to_integer: :environment do
+    Comment.where(old_status: Comment.statuses.keys).each do |comment|
+      comment.update(status: Comment.statuses[comment.old_status])
+    end
+
+    Comment.where(old_status: Comment.statuses.values.map(&:to_s)).each do |comment|
+      comment.update(status: comment.old_status.to_i)
+    end
+  end
+
+  task convert_integer_status_to_string: :environment do
+    Comment.all.each do |comment|
+      comment.update(old_status: comment.status)
+    end
+  end
```

Comment:
Those tasks and the ones above are differ only by a model. All the rest is the same. You need only one file. Check how to pass params to a rake task and send the model as a param.